superset/assets/javascripts/profile/components/CreatedContent.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import moment from 'moment';
+import TableLoader from './TableLoader';
+
```

Comment:
oh interesting, the linter didn't trigger because it's unaware of what's hapenning inside the backticks